src/EnergyPlus/HybridEvapCoolingModel.hh

```diff
@@ -0,0 +1,347 @@
+
+#ifndef HybridEvapCoolingModel_hh_INCLUDED
+#define HybridEvapCoolingModel_hh_INCLUDED
+#include <iostream>  
+//#include <fmiPlatformTypes.h>
```

Comment:
Commented out code should be removed unless it is crucial to the understanding of the functional code.